### ISSUE TITLE
Swiper: Boss - Sound 

### ISSUE THREAD

**Issue by Shusama** [OpenAngelArena/oaa#2126]
# Boss Swiper
We have several bosses already implemented which you should feel free to use as inspiration/reference. Also you can reference some other open source custom games on Github which have boss fights including Second Day of Frostivus where you will find our IO boss (designed by Yahnich). Files should follow the structure already established in the components subfolders (such as /abilities/, /boss/, etc).

### See also:
Swiper: Boss - Model/Animations/Icons 
Swiper: Boss - Sound 
Swiper: Boss - Particles

[Bug Bounty Guideline](https://docs.google.com/document/d/1Tpm0j1ekVc6X-LRlF_6Q81RMGVlOOO5EBvSf_xAQk4I/edit)

## Sound
Since most of his abilities are sword swings they should also sound like those. Have a different sound for the abilities hitting something elsewise something like simple sword swings through air.

**Comment by hesamironman** [comment]
@Shusama @l34Um1 I claim this issue. Please tell me how to send it to you guys??

**Comment by chrisinajar** [comment]
Create a pull request containing all of the sounds and the `.vsndevts` file.

If you need help with the later, ask in discord.

**Comment by chrisinajar** [comment]
This ticket is basically pure data entry, the generic material-specific sounds already exist in base dota.

**Comment by l34Um1** [comment]
Uh no, this ticket is about creating custom sound effects Chris.

**Comment by hesamironman** [comment]
look I can create them in mp3 or wav format. I have no idea abot .vsndevts files :/

**Comment by hesamironman** [comment]
@l34Um1 I already have some sounds for Electrician. Can I create a pull request and send them there?

**Comment by chrisinajar** [comment]
Indeed, dota already has swish, strike, and clang noises with every combination of materials. Additionally the data entry format supports speed, volume, and pitch shift options. One can easily finish this ticket without creating any new sound files. Custom sounds are nice-to-have, pitch shifting and playing sounds over each other are how dota does most of its non-skill sound effects, and even many skills are 1 custom sound played over a generic one or purely composed of generic sounds.

the `.vsndevts` file is what describes to dota how to play the sounds, including the aforementioned pitch/volume/speed settings, and also gives all the files name. It supports both wav and mp3, and compiles the audio file into dota's proprietary format.

**Comment by chrisinajar** [comment]
most of the work for this ticket will be making small tweaks to the vsndevts to get the sounds to feel nice in engine. I did this process for sohei last time, it's a very iterative but quite intuitive process.

**Comment by hesamironman** [comment]
@chrisinajar OK. So I just create a pull request and send all wav format sound effects there. I already have some sounds for Electrician.

**Comment by chrisinajar** [comment]
Yes, create a pull request containing all of the files in the right spots and also the vsndevts file to describe them all. The sounds wont play without the vsndevts file. I *highly* recommend looking at sohei for an example.

**Comment by hesamironman** [comment]
@chrisinajar I'm a musician and I just joined github like 2 days ago to help you guys with sound "issues"> To be honest Chris, I have no idea about vsndevts files!

**Comment by chrisinajar** [comment]
You need to get onto discord and also be able to run the game locally in tools mode, otherwise you have no way of knowing how it sounds in-engine.

https://github.com/OpenAngelArena/oaa/blob/master/docs/setup/install.md

**Comment by hesamironman** [comment]
@chrisinajar Alright I'll check it out. Thanks

**Comment by mikibun** [comment]
@hesamironman has this been finished? :)

**Comment by hesamironman** [comment]
@mikibun I've  finished both this and electrician, but as I said I just gave up, because I don't know about vsndevts files. If it's helpful, I can recreate them and send them but I have no idea about that particular format. I can only make in wav or mp3 :)

**Comment by mikibun** [comment]
@hesamironman I think people might be able to convert it for you, I'll ask around, so hold fire.

**Comment by hesamironman** [comment]
@mikibun Alright :)

**Comment by hesamironman** [comment]
I have Swiper and Electrician sounds! Can someone help me with creating a pull request please?
